javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,103 @@
+// FreeCodeCamp user: https://www.freecodecamp.org/catalina-tech
+
+
+
+// Age-ify (A future age calculator)
+
+let yearOfBirth;
+yearOfBirth = 1988;
+console.log(typeof yearOfBirth);
+
+let yearFuture;
+yearFuture = 2028;
+console.log(typeof yearFuture);
+
+let age = yearFuture - yearOfBirth;
+console.log("You will be" + " " + age + " " + "in" + " " + yearFuture); 
+
+
+
+
+//Goodboy-Oldboy (A dog age calculator)
+
+let dogYearOfBirth;
+dogYearOfBirth = 2017;
+
+let dogYearFuture;
+dogYearFuture = 2027;
+
+//Human years
+let dogYear = dogYearFuture - dogYearOfBirth;
+
+//Dog Years
+let dogYearOld = dogYear*7;
+
+let shouldShowResultInDogYears = false;
+
+if (shouldShowResultInDogYears) {
+    console.log("Your dog will be" + " " + dogYearOld + " " + "dog years old in " + dogYearFuture);    
+} else {
+    console.log("Your dog will be" + " " + dogYear + " " + "human years in " + dogYearFuture);  
```

Comment:
Same comment as before, you can combine the spaces with the other strings:
```suggestion
    console.log("Your dog will be " + dogYear + " human years in " + dogYearFuture);  
```